FiveCalls/FiveCalls/ScheduleReminders.swift

```diff
@@ -0,0 +1,285 @@
+//
+//  ScheduleReminders.swift
+//  FiveCalls
+//
+//  Created by Christopher Selin on 9/8/23.
+//  Copyright © 2023 5calls. All rights reserved.
+//
+
+import SwiftUI
+
+let USE_SHEET = true
+
+struct ScheduleReminders: View {
+    @AppStorage("reminderEnabled") var remindersEnabled = false
+    @Environment(\.dismiss) var dismiss
+
+    @State var existingSelectedTime = Date.distantPast
+    @State var selectedTime = Date.distantPast
+    @State var existingSelectedDayIndices = [Int]()
+    @State var selectedDayIndices = [Int]()
+    @State var shouldShake = false
+    
+    var body: some View {
+        if !USE_SHEET {
```

Comment:
Depending on whether we use a sheet or not, a lot of this duplicated code can be removed.

Reply:
@nickoneill did you want to still keep the same presentation (sheet) for this for now? I already have it working so I can just remove my other push presentation implementation.